test/SaveHandler/DbTableGatewayTest.php

```diff
@@ -175,41 +175,6 @@ public function testDestroyReturnsTrueWhenSessionIsDeleted()
         $this->assertTrue($result);
     }
 
-    public function testExpiredSessionDoesNotCauseARecursion()
```

Comment:
@michalbundyra Since there is no recursion happening anymore this is impossible. Adding a test for recursion where recursion is not happening anymore would only confuse developers.

Reply:
@rjd22 Ok, fine. There is no harm of removing it and I see your point. Thanks